setup.py

```diff
@@ -13,23 +13,28 @@
     author_email="stefanik.m@fi.muni.cz",
     url="https://gitlab.fi.muni.cz",
     license="MIT",
-    packages=["pv211_utils", "pv211_utils.trec", "pv211_utils.cranfield", "pv211_utils.arqmath", "pv211_utils.systems"],
+    packages=["pv211_utils", "pv211_utils.trec", "pv211_utils.cranfield", "pv211_utils.arqmath",
+              "pv211_utils.beir", "pv211_utils.systems"],
     package_dir={"pv211_utils": "pv211_utils"},
     include_package_data=True,
     zip_safe=True,
     setup_requires=[
         "setuptools",
     ],
     install_requires=[
+        "beir",
         "gspread",
         "oauth2client",
         "google",
         "tqdm",
         "ijson",
         "ipython",
+        "ipywidgets",
         "gensim",
         "sentence_transformers",
-        "torch"
+        "sympy",
+        "torch",
+        "scikit-learn"
```

Comment:
Pin versions of all libraries.

Reply:
@VojtechKalivoda pinned versions in https://github.com/MIR-MU/pv211-utils/pull/8/commits/9af8d8e9dc643bcdeb030c35a58c8ef230c884ca by adding file `requirements.txt`. We can use [pip's `parse_requirements()` function][1] to sync `setup.py`'s `install_requires` with `requirements.txt`.

 [1]: https://github.com/MIR-MU/pine/blob/cd29184575fd37d644066de541862b32470157b0/setup.py#L14-L17